internal/controllers/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -279,9 +279,15 @@ func (r *Reconciler) reconcile(ctx context.Context, logger logr.Logger, cluster
 			Message:  message,
 		})
 
+		eventType := corev1.EventTypeWarning
```

Comment:
Should we even publish an event in that case?

Reply:
In this case, the event doesn't seem to make any sense, I'll fix it.